system_tests/system_tests.go

```diff
@@ -255,22 +256,22 @@ CONSOLE_LOG=new`
 
 		It("persists messages after pod deletion", func() {
 			By("publishing a message", func() {
-				err := rabbitmqPublishToNewQueue(hostname, port, username, password)
+				err := publishToNewQueue(hostname, port, username, password)
```

Comment:
Small: Rename to `publishToQueue`. I am assuming we are consuming from the same queue in the next `By` block.